rxandroidble/src/main/java/com/polidea/rxandroidble2/exceptions/BleDisconnectedException.java

```diff
@@ -12,26 +14,38 @@
  */
 public class BleDisconnectedException extends BleException {
 
+    /**
+     * Set when the state is not available, for example when the adapter has been switched off.
+     */
+    public static final int UNKNOWN_STATE = -1;
```

Comment:
Shouldn't it be `status`? And maybe `STATUS_UNAVAILABLE`?

Reply:
You're right.